pkgs/fc/ceph/src/fc/ceph/maintenance/nautilus.py

```diff
@@ -218,7 +242,20 @@ def enter(self):
             # 69 signals to postpone the maintenance, triggering a leave in fc-agent
             sys.exit(69)
 
+        # Prohibit OSD traffic by marking them down and flagging them to
+        # not automatically return.
+        try:
+            host_buckets, osd_ids = get_host_crush_buckets()
+            run.ceph("osd", "set-group", "noup", *sorted(host_buckets))
```

Comment:
Interestingly, that's yet another command not listed in the ceph man page. I had to resort to docs of more recent versions, but apparently this already works in Nautilus.

Reply:
Yeah. I found this in a mailing list archive and had to look through the source code as someone mentioned it being there on Nautilus already. Luck us, I guess ... ;)